src/site/apt/license_def.apt.vm

```diff
@@ -29,7 +29,9 @@ How to define licenses in Apache Rat
  scope of this document.  In this document we will address how to define licenses in the default XML format.
  
  The XML document has a root node named "rat-config" which comprises 4 major sections: "Families", 
- "Licenses", "Approved" and "Matchers".
+ "Licenses", "Approved" and "Matchers".  There is an {{{./apache-rat/rat.xsd}}XSD} that can be used to validate
```

Comment:
@Claudenw is there a technical reason for each sentence to start with 2 spaces? (or is it just a formatter setting of yours)